@utahdts/utah-design-system/react/components/buttons/Tag.jsx

```diff
@@ -0,0 +1,155 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import formElementSizesEnum from '../../enums/formElementSizesEnum';
+import joinClassNames from '../../util/joinClassNames';
+import handleEvent from '../../util/handleEvent';
+import RefShape from '../../propTypesShapes/RefShape';
+import IconButton from './IconButton';
+
+const propTypes = {
+  // most often is the title of the tag, but can also contain most anything
+  children: PropTypes.node.isRequired,
+  // modify your tag via classname like 'tag--primary' and other modifiers found in the tag.scss
+  className: PropTypes.string,
+  // a ref to attach to the actual DOM <button> or <span> element
+  innerRef: RefShape,
+  // an icon for the left/right side
+  iconLeft: PropTypes.node,
+  iconRight: PropTypes.node,
+  // the tag id
+  id: PropTypes.string,
+  isClearable: PropTypes.bool,
+  // tag isDisabled state
+  isDisabled: PropTypes.bool,
```

Comment:
what does it mean to have a `disabled` tag?

Reply:
Will split into two components.